### DISCUSSION TITLE
Fullscreen / Focus Mode for richtext editor

### DISCUSSION THREAD

**Discussion by mhddngs** [payloadcms/payload#9840] [Feature Requests & Ideas]
### What
Add a toggle to enter fullscreen / immersive / focus mode for the richtext editor.

### Why
Currently the editing experience can feel a bit cramped and distracting, especially for non-technical editors. The flexibility of Lexical is a big selling point to make dynamic documents and it would be cool to be able to take full advantage of the screen real estate to craft those documents and focus up. 

### Example
A toggle over the lexical editor toolbar would be a straightforward approach as pictured below, but there might also be an opportunity to bring the "focus" mode to a higher layer. For instance, maybe Content, Meta, and SEO should be included as tabs in the fullscreen mode in which case you may want to reposition the fullscreen toggle to a nearby area and not on the lexical toolbar.

![Frame 2](https://github.com/user-attachments/assets/68a44227-caed-4004-ac9c-09cb16a26a1c)

**Comment by rilrom** [comment]
I've built a plugin to provide this functionality until the payload team implement it.

https://github.com/rilrom/payload-bites/tree/main/packages/fullscreen-editor

Please take it for a spin and let me know if you run into any issues!
